app/src/main/java/org/projectbuendia/client/ui/chart/PatientChartActivity.java

```diff
@@ -271,6 +281,28 @@ public void onPageFinished(WebView view, String url) {
                 mController.onAddTestResultsPressed();
             }
         });
+
+        initChartMenu();
+    }
+
+    private void initChartMenu() {
+        List<Chart> charts = mController.getCharts();
+        String[] menuArray = new String[charts.size()];
+        int i = 0;
```

Comment:
If you have to maintain the loop variable anyway, you may as well use the 
```java
for (int i = 0; i < charts.size(); i++) {
```
form.